defaults/main.yml

```diff
@@ -11,6 +11,9 @@ gitlab_runner_wanted_version: latest
 # This variable should not be modified usually as it depends on the gitlab_runner_wanted_version variable
 gitlab_runner_wanted_tag: "{{ 'latest' if gitlab_runner_wanted_version == 'latest' else ('v' + gitlab_runner_wanted_version) }}"
 
+# If a different partition or disk is used under Windows
+gitlab_runner_install_directory: c:/gitlab-runner/
```

Comment:
I guess your problem is to not being able to override this particular variable? If so, this is true for all variables which are included with `include_vars`. Instead of moving variables from the platform specific files to this general one and pollute it that way, we should think of a (clever) way to make platform-specific variables also overridable.

Reply:
I don't think this is a general problem for all variables in this file, as they only represent the “default”. But the location where I want to install something should be definable from my point of view, it would also be possible that Windows has no C: drive at all. Since Ansible has variable precedence, I don't think there will be a clever way around this, or maybe it shouldn't be implemented. 

https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

It's also not impossible to override the platform specific variables like the download URL, but in this case I have to override them for each host, which is not possible once the precedence is higher than the host_vars.

@guenhter sorry for the late reply, the notification passed me by :/